Source/UI/Views/TextView.swift

```diff
@@ -4,14 +4,19 @@
 //
 
 class TextView: UITextView, Themeable {
```

Comment:
Can this TextView also be used for the terms screen (since they show links as well)?

Reply:
Nevermind, didn't realize this was there already 🤦‍♂️